### PR TITLE
README: pip: add `wheel`, `pip >= 19.1.1`

### PR DESCRIPTION
**Pull request by tj90241**
* `ansible` requires `wheel` to be installed, and will
    fail to install correctly (even when it is in the same
    `pip install ...` invocation as `wheel`).

  * Ensure users have `pip>=19.1.1` to avoid issues when
    installing `cryptography>3.4` as part of `pycryptodome`,
    etc.

Additionally, reflect the semantics of using `pip>=19.1.1`
but without `--upgrade` in the `rally` recipe.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ Create a Python virtual environment (virtualenv) and activate it
 ```shell
 python3 -mvenv venv
 source venv/bin/activate
+pip install wheel 'cryptography<3.4'
```
Comment (comay) [inline README.md:56]: This is fine but was the workaround that @mboruta1 suggested with Rally of updating `pip` make more sense?

Comment (mboruta1) [reply README.md:56]: For reference my explanation for doing so can be found [here](https://github.com/bloomberg/chef-bcpc/pull/1784#issue-576482643). It is also the solution recommended by the package itself (see [this](https://github.com/pyca/cryptography/blob/main/CHANGELOG.rst#342---2021-02-08)).

Comment (tj90241) [reply README.md]: Tested suggestion and as expected, works great -- thanks!

**Review by mboruta1 — APPROVED** [review]
Looks good to me

**Review by comay — APPROVED** [review]

**Pull request merged**